src/WorkBC.Data/Model/JobBoard/NocCodeSSOT.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace WorkBC.Data.Model.JobBoard
+{
+    public class NocCodeSSOT
```

Comment:
Since this class is only used in the migration, I suggest to move it to the migration file.

Reply:
Moved the class to the migration script.